MacaoProject/Macao/Models/Player.cs

```diff
@@ -8,9 +8,7 @@ namespace Macao
 {
     public class Player
```

Comment:
the player does not know about his cards. if i want at any time to see what cards the player has i cannot see them. you should store them here. Also place card method should place a card from the players hand let's say so from his own cards which at the moment you do not have